combat-trainer.lic

```diff
@@ -1177,6 +1177,9 @@ class TrainerProcess
 
     @forage_item = settings.forage_item
     echo("  @forage_item: #{@forage_item}") if $debug_mode_ct
+
+    @analyze_retry_count = settings.analyze_retry_count
```

Comment:
The name of this setting is generic enough that I would expect it to affect all types of analyze. What do you think?